utils/clevis/patches/001-no-old-cryptsetup.patch

```diff
@@ -0,0 +1,24 @@
+--- a/src/luks/tests/meson.build	2023-09-21 18:25:28.574733087 +0200
++++ b/src/luks/tests/meson.build	2023-09-21 18:25:48.804764217 +0200
```

Comment:
Why do we need to have this patch? Who is the author? Is this backport from upstream repository?

Reply:
I have had error like below: 
 "ERROR: Entry OLD_CRYPTSETUP not in configuration data."

Reply:
Did you discussed it with upstream?

Reply:
No. I'll investigate the problem and will try to disable all tests during build process.

Reply:
Anything new?